cpp/benchmarks/CMakeLists.txt

```diff
@@ -24,14 +24,14 @@ endfunction(ConfigureBench)
 
 include_directories("${CUB_INCLUDE_DIR}"
                     "${THRUST_INCLUDE_DIR}"
+                    "${JITIFY_INCLUDE_DIR}"
                     "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"
                     "${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/include"
                     "${CMAKE_SOURCE_DIR}"
                     "${CMAKE_SOURCE_DIR}/src"
                     "${CMAKE_SOURCE_DIR}/thirdparty/dlpack/include"
-                    "${CMAKE_SOURCE_DIR}/thirdparty/jitify"
-                    "${CMAKE_SOURCE_DIR}/thirdparty/libcudacxx/include"
+                    "${LIBCUDACXX_INCLUDE_DIR}"
```

Comment:
May want to move this above the CUDA toolkit includes as well just in case it ships in the toolkit in the future.